api/DOMRectList.json

```diff
@@ -0,0 +1,148 @@
+{
+  "api": {
+    "DOMRectList": {
+      "__compat": {
+        "spec_url": "https://drafts.fxtf.org/geometry/#DOMRectList",
+        "support": {
+          "chrome": {
+            "version_added": "2"
```

Comment:
https://mdn-bcd-collector.appspot.com/tests/api/DOMRectList papers over this in order to test the attributes on an instance, but the original name for `DOMRectList` was `ClientRectList`. Can you compare to the api.DOMRect entry and add similar ranges? I would expect the version ranges are the same, but it's best to test to confirm.